Please fix the compiling error

```
In file included from /home/travis/build/cocos2d/cocos2d-x/cocos/2d/CCGrid.cpp:29:0:
/home/travis/build/cocos2d/cocos2d-x/cocos/base/ccUtils.h:180:45: error: ‘Data’ does not name a type
     CC_DLL std::string getDataMD5Hash(const Data &data);
```

--- a/cocos/base/ccUtils.h
+++ b/cocos/base/ccUtils.h
@@ -29,6 +29,7 @@ THE SOFTWARE.
 #include <string>
 #include "2d/CCNode.h"
 #include "base/ccMacros.h"
+#include "base/CCData.h"
 
 /** @file ccUtils.h
 Misc free functions

